docs/getting-started/github-apps.rst

```diff
@@ -45,47 +44,117 @@ sync with the linked GitHub repository.
 How the Dockstore GitHub App works
 ----------------------------------
 
-With the Dockstore GitHub App, the synchronization is done automatically. When
+With the Dockstore GitHub App installed, the synchronization is done automatically. When
 you add a new branch or release of a workflow on GitHub, Dockstore is notified,
 and Dockstore updates its copy of the workflow. For example, After publishing a new release
 of a workflow on GitHub, a new version of the workflow will be present in
 Dockstore shortly afterwards.
 
-For this to work, a ``/.dockstore.yml`` file is required in each GitHub repository you want
+For this to work, a ``/.dockstore.yml`` file is required in the root directory of each GitHub repository you want
 to associate with a workflow on Dockstore. A template for both workflows and services are shown below,
-along with explanations for each field. For every branch on GitHub that has one of these files, a corresponding entry
+along with explanations for each field. For every branch/release on GitHub that has one of these files, a corresponding entry
 will be made on Dockstore.
 
 Workflow YML File
 ++++++++++++++++++
+For a workflow, the ``/.dockstore.yml`` has the following general structure
 
 .. code:: yaml
 
    version: 1.2
    workflows:
-      - name: aligner
-        subclass: CWL
-        primaryDescriptorPath: /Dockstore.cwl
-        testParameterFiles:
-            - /test/dockstore.cwl.json
+      - name: <String>
+        subclass: <CWL | WDL | NFL | GXFORMAT2>
+        primaryDescriptorPath: <String>
+        testParameterFiles: <String Array>
 
 version
     The version of the .dockstore.yml schema. Currently at 1.2.
 workflows
     An array of workflows. Each element corresponds to a workflow on Dockstore.
-name
+name (optional)
     The optional workflow name that is used to uniquely identify workflows in repositories with multiple workflows.
+    **Each workflow listed must have a unique (or no) name.**
 subclass
-    The descriptor language used for the workflow. Supported values include CWL, WDL, and NFL.
+    The descriptor language used for the workflow. Supported values include CWL, WDL, NFL (Nextflow), and GXFORMAT2 (Galaxy).
 primaryDescriptorPath
-    The path to the primary descriptor file in the Git repository
-testParameterFiles
-    An array of paths to test parameter files in the Git repository.
+    The absolute path to the primary descriptor file in the Git repository
+testParameterFiles (optional)
```

Comment:
I think this key is required, although it is not required to have any values. We should probably make it optional. If not, we should change this. Can you please log a ticket if you're going to keep the doc like this?

Reply:
I believe I have tested without the testParameterFiles key and it works.